test-support/helpers/ember-tooltips.js

```diff
@@ -0,0 +1,90 @@
+import Ember from 'ember';
+
+const getTooltipFromBody = function($body, selector='.ember-tooltip, .ember-popover', oneTooltipMustExist=true) {
+  // we have to .find() tooltips from $body because sometimes
+  // tooltips and popovers are rendered as children of <body>
+
+  if ($body.prop('tagName') !== 'BODY') {
+    throw Error('the first parameter must be $body');
+  }
+
+  let $tooltip = $body.find(selector) ;
+
+  if (oneTooltipMustExist) {
+    if (!$tooltip.hasClass('ember-tooltip') && !$tooltip.hasClass('ember-popover')) {
+      throw Error(`getTooltipFromBody(): no $(${selector}) in $body found`);
+    } else if ($tooltip.length > 1) {
+      throw Error('getTooltipFromBody(): Multiple tooltips where found. Please provide an {option.selector = ".specific-tooltip-class"}')
+    }
+  }
+
+  return $tooltip;
+}
+
+export function triggerTooltipEvent($element, type, options={}) {
+
+  // TODO why do we allow focusin? why not just focus?
+  const approvedEventTypes = ['mouseenter', 'mouseleave', 'click', 'focus', 'focusin', 'blur'];
+  if (approvedEventTypes.indexOf(type) == -1) {
+    throw Error(`only approvedEventTypes will affect the tooltip target. You used ${type}`);
```

Comment:
1) Should this be a warning? 2) Should we echo out the `approvedEventTypes` or provide a link for further information? Maybe this should be included in the documentation.